sql/01-enable_versioning.sql

```diff
@@ -228,6 +228,11 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql;
 
-DROP EVENT TRIGGER IF EXISTS _ver_abort_on_drop_of_versioned_table;
-CREATE EVENT TRIGGER _ver_abort_on_drop_of_versioned_table
+-- This is only needed on databases which ever installed a
```

Comment:
Hi @strk , I there was no deployment done with dev version. Please drop this command